src/utils/validateName.ts

```diff
@@ -0,0 +1,7 @@
+// prettier-ignore
+export default function validateName(name: string): string | void {
```

Comment:
non alphanumeric values will be valid when they shouldnt. eg: "ab1$d". An extra check is needed